sdks/java/core/src/main/java/org/apache/beam/sdk/util/ReduceFnRunner.java

```diff
@@ -108,7 +109,7 @@
   /**
    * Track which windows are still active and which 'state address' windows contain state
    * for a merged window.
-   *
+   * <p>
```

Comment:
Did you notice that this was needed? The `<ul>` is already a block-level element.